### PR TITLE
use Odra 0.6.0

### PR DESCRIPTION
**Pull request by zie1ony**

### CODE REVIEW

File: dao/src/core_contracts/reputation/balances.rs [hunk 1]
```diff
@@ -1,12 +1,13 @@
-use std::{
-    collections::BTreeMap,
-    ops::{AddAssign, SubAssign},
-};
-
+// use std::{
```
Comment (kubaplas) [inline dao/src/core_contracts/reputation/balances.rs:1]: delete this?

File: dao/src/modules/repository.rs [hunk 2]
```diff
@@ -25,7 +30,7 @@ pub struct Record {
 #[odra::module(events = [ValueUpdated])]
 pub struct Repository {
     pub storage: Mapping<String, Record>,
-    pub keys: List<String>,
+    pub keys2: List<String>,
```
Comment (kubaplas) [inline dao/src/modules/repository.rs:33]: rename to keys

File: dao/src/core_contracts/reputation/balances.rs (hunk 1 above)
Comment (zie1ony) [reply dao/src/core_contracts/reputation/balances.rs]: done

File: dao/src/modules/repository.rs (hunk 2 above)
Comment (zie1ony) [reply dao/src/modules/repository.rs]: changed to `all_keys`

File: tmp/admin_contract_wasm.rs [hunk 3]
```diff
@@ -0,0 +1,831 @@
+#![no_main]
```
[the remark this answers is not in the document]
Comment (zie1ony) [reply tmp/admin_contract_wasm.rs]: removed

**Review by kubaplas — APPROVED** [review]

**Pull request merged**